.github/workflows/pushCatalogImages.yaml

```diff
@@ -19,10 +19,28 @@ jobs:
 
   build:
     name: Build
-    runs-on: ubuntu-latest
-    
+    runs-on: ubuntu-20.04
+
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
+      with:
+        path: go/src/github.com/eclipse/${{ github.repository }}
+        fetch-depth: 0
+
+    - name: Prepare new nightly Olm bundle
+      run: >-
+        export GOPATH="${GITHUB_WORKSPACE}/go" &&
+        echo "GOPATH: ${GOPATH}" &&
+        export GITHUB_WORKSPACE="${GOPATH}/src/github.com/eclipse/${{ github.repository }}" &&
+        cd ${GITHUB_WORKSPACE} && "${GITHUB_WORKSPACE}/.github/action_scripts/provideNewNightlyBundle.sh"
+
+    - name: Commit new nightly Olm bundle
```

Comment:
WDYT about adding a custom message in the commit?

Reply:
It's a good idea, but I think, we can improve it in the next pr's.